pennylane/__init__.py

```diff
@@ -176,36 +177,39 @@ def device(name, *args, **kwargs):
     * :mod:`'default.qubit' <pennylane.devices.default_qubit>`: a simple
       state simulator of qubit-based quantum circuit architectures.
 
-    * :mod:`'default.gaussian' <pennylane.devices.default_gaussian>`: a simple simulator
-      of Gaussian states and operations on continuous-variable circuit architectures.
+    * :mod:`'default.mixed' <pennylane.devices.default_mixed>`: a mixed-state
+      simulator of qubit-based quantum circuit architectures.
 
-    * :mod:`'default.qubit.tf' <pennylane.devices.default_qubit_tf>`: a state simulator
-      of qubit-based quantum circuit architectures written in TensorFlow, which allows
-      automatic differentiation through the simulation.
+    * ``'lightning.qubit'``: a more performant state simulator of qubit-based
+      quantum circuit architectures written in C++.
 
-    * :mod:`'default.qubit.autograd' <pennylane.devices.default_qubit_autograd>`: a state simulator
-      of qubit-based quantum circuit architectures which allows
-      automatic differentiation through the simulation via python's autograd library.
+    * :mod:`'default.qutrit' <pennylane.devices.default_qutrit>`: a simple
+      state simulator of qutrit-based quantum circuit architectures.
+
+    * :mod:`'default.gaussian' <pennylane.devices.default_gaussian>`: a simple simulator
+      of Gaussian states and operations on continuous-variable circuit architectures.
 
     Additional devices are supported through plugins — see
     the  `available plugins <https://pennylane.ai/plugins.html>`_ for more
-    details.
+    details. To list all currently installed devices, run
+    :func:`qml.about <pennylane.about>`.
 
     Args:
         name (str): the name of the device to load
         wires (int): the number of wires (subsystems) to initialise
-            the device with
+            the device with. Note that this is optional for certain
+            devices, such as ``default.qubit``
 
     Keyword Args:
         config (pennylane.Configuration): a PennyLane configuration object
```

Comment:
Any idea what devices take a `Configuration`? I don't think I've ever actually touched this class.

Reply:
no clue, I have never touched it either, but I think it's still technically true.